### PR TITLE
Add docs for new Giskard Docker template in HuggingFace Spaces

### PR DESCRIPTION
**Pull request by luca-martial**

### CODE REVIEW

File: docs/hub/spaces-sdks-docker-giskard.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+Once inside the app, start by creating a new project from the welcome screen.
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-giskard-create-project.png"/>
+</div>
+
+## Generate a Giskard Space token
+
+The Giskard Space token is used to establish communication between the environment where your AI models are running and
```
Comment (Inokinoki) [inline docs/hub/spaces-sdks-docker-giskard.md]: I do not find the HF term for this token. Let's align with them if they asked.

File: docs/hub/spaces-sdks-docker-giskard.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+<div class="flex justify-center">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-giskard-create-project.png"/>
+</div>
+
+## Generate a Giskard Space token
+
+The Giskard Space token is used to establish communication between the environment where your AI models are running and 
+the Giskard HuggingFace Space. If you've set the **Visibility** of your Space to **Private**, you will need to provide a HuggingFace 
+access token to generate the Giskard Space Token and establish a communication. To do so, follow the instructions
```
Comment (Inokinoki) [inline docs/hub/spaces-sdks-docker-giskard.md]: Let's build the link between the semantic `Giskard Space Token` and the `HF-TOKEN` used below.

HF users may be confused by `HF-TOKEN` and their [HF User Access token](https://huggingface.co/docs/hub/main/en/security-tokens).

Comment (luca-martial) [reply docs/hub/spaces-sdks-docker-giskard.md]: Ah ok, so what should we name each of them?

Comment (Inokinoki) [reply docs/hub/spaces-sdks-docker-giskard.md]: I think either we call it SPACE_TOKEN, or we can explicitly say HF_TOKEN is the Giskard Space Token

Comment (Inokinoki) [inline docs/hub/spaces-sdks-docker-giskard.md]:
```suggestion
access token to generate the Giskard Space Token (`HF-TOKEN` in the following) and establish a communication. To do so, follow the instructions 
```

**Pull request merged**